src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -3019,6 +3019,14 @@ checkNullable:
                 closeParen = closeParen.AddLeadingSyntax(unexpected)
             End If
 
+            If elementBuilder.Count < 2 Then
```

Comment:
>If elementBuilder.Count < 2 Then [](start = 12, length = 32)

Consider adding an assert that we never have an empty elementBuilder at this point.